third_party/move/tools/move-mutator/src/compiler.rs

```diff
@@ -82,13 +88,18 @@ pub fn generate_ast(
         flags,
         &config.compiler_config.known_attributes,
     )
-    .set_interface_files_dir(interface_files_dir.to_str().unwrap_or("").to_string())
+    .set_interface_files_dir(
+        interface_files_dir
+            .to_str()
+            .expect("Output path contains invalid characters.")
```

Comment:
According to the [convention](https://doc.rust-lang.org/std/error/index.html#common-message-styles), lowercase and dot at the end.
```suggestion
            .expect("output path contains invalid characters")
```

Reply:
Done.